### ISSUE TITLE
Need to update our deployment to Node20

### ISSUE THREAD

**Issue by bennyvasquez** [AlmaLinux/almalinux.org#469]
Getting a warning on our automated testing deployments:

`Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3, peaceiris/actions-hugo@v2, cloudflare/pages-action@v1. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.`

**Comment by jonathanspw** [comment]
Monitoring cloudflare/pagesaction via https://github.com/cloudflare/pages-action/issues/117 and peaceiris/actions-hugo via https://github.com/peaceiris/actions-hugo/issues/642

**Comment by bennyvasquez** [comment]
As Jonathan noted, we're waiting on 

- [ ] https://github.com/cloudflare/pages-action/issues/117
- [ ] https://github.com/peaceiris/actions-hugo/issues/642

**Comment by codyro** [comment]
Workaround in case this doesn't get updated anytime soon: https://github.com/cloudflare/pages-action/issues/117#issuecomment-2053610761

**Comment by bennyvasquez** [comment]
This is already done.
